src/ScanPatron/ScanForm.js

```diff
@@ -23,6 +23,14 @@ import { useReadingRoom, useProfilePicConfigForTenant } from '../hooks';
 
 import css from './ScanForm.css';
 
+const NoReadingRoom = () => (
+  <Row>
+    <Col xs={10} className={css.noReadingRooms}>
+      No reading rooms defined for the current service point
```

Comment:
Please add this text to translations

Reply:
As discussed, I have added a translation for this text.